### DISCUSSION TITLE
Berry log to file

### DISCUSSION THREAD

**Discussion by araczkowski** [arendst/Tasmota#19641] [Q&A] [answered]
Hi, 
Is it possible to write Tasmota logs to file?
Can berry do this?

🙏 Thanks

**Comment by sfromis** [comment, marked as answer]
Yes, you can access logs from Berry, and create code to write to a file.
https://tasmota.github.io/docs/Berry/#tasmota_log_reader-class

**Comment by araczkowski** [comment]
thanks @sfromis I will try!
